dev/package.sh

```diff
@@ -4,17 +4,28 @@ set -eux
 
 CURRENT_DIR=$(cd "$(dirname "$BASH_SOURCE")"; pwd)
 GLUTEN_DIR="$CURRENT_DIR/.."
-THIRDPARTY_LIB="$GLUTEN_DIR/package/target/thirdparty-lib"
+DISTDIR="$GLUTEN_DIR/dist"
+THIRDPARTY_LIB="$DISTDIR/thirdparty-lib"
```

Comment:
I guess there are some docs should be updated

Reply:
Thanks, there seems to be no more documentation, just a commented script..

![image](https://github.com/apache/incubator-gluten/assets/17894939/3324edf7-5624-4530-8e33-aea4e3f4bdb9)